src/components/dashboard/Dashboard.js

```diff
@@ -497,6 +505,46 @@ const Dashboard = props => {
 
   const avatarSource = avatar || unknownProfile
 
+  const memoFeedList = useMemo(() => {
+    return (
+      <FeedList
+        data={feeds}
+        handleFeedSelection={handleFeedSelection}
+        initialNumToRender={PAGE_SIZE}
+        onEndReached={nextFeed}
+        onEndReachedThreshold={0.7}
+        windowSize={7}
```

Comment:
why did you choose thershold 0.7 and windowsize 7? add explanation in code comment

Reply:
done